src/main/resources/public/locales/en/dc-exporter.json

```diff
@@ -83,5 +83,20 @@
             "tags": "Tags",
             "resource_name": "Resource Name"
         }
+    },
+
+    "css_class": {
```

Comment:
It seems weird to have that in a translation file. What about putting it in the React component (it is just a map) ?